fcli-core/fcli-aviator/src/main/java/com/fortify/cli/aviator/ssc/cli/cmd/AviatorSSCAuditCommand.java

```diff
@@ -42,98 +45,87 @@ public class AviatorSSCAuditCommand extends AbstractSSCJsonNodeOutputCommand imp
     @Option(names = {"--tag-mapping"}, required = false, description = "Tag Mapping") private String tagMapping;
 
     private static final Logger LOG = LoggerFactory.getLogger(AviatorSSCAuditCommand.class);
-    private String auditAction;
+
+    private static final ThreadLocal<JsonNode> cachedResult = new ThreadLocal<>();
```

Comment:
As mentioned before: As per fcli best practices, command classes shouldn’t use instance fields to hold state; only instance fields annotated with Picocli annotations like `@Mixin` or `@Option` are allowed. Although not explicitly stated, storing state in static fields is even worse (and `ThreadLocal` doesn't help); it's not only considered a violation of fcli best practices, but may cause major bugs, especially when used in combination with fcli actions.

As a simple example, see output of the `bulkaudit.yaml` action developed by @fransvanbuul:
![image](https://github.com/user-attachments/assets/4f39c087-77d8-49c5-9310-36f67cd0e1eb)